### ISSUE TITLE
settings menus can be cut off add scroll?

### ISSUE THREAD

**Issue by jjfmz42015** [punesemu/puNES#83]
So one of my setups i use screen resolution 1280x720.

I do this because its a media pc in my living room on a 60 inch tv 15 to 20 feet away.

At a distance higher resolutions make things way small

puNES config menus get cut off on this low a resolution.

Can maybe scroll bars be added to allow scrolling? 

Example "input config" and Palette config

**Comment by punesemu** [comment]
Why did you close the issue? If it's because I didn't answer yesterday, I have little free time during the week, I don't connect often.

**Comment by jjfmz42015** [comment]
> Why did you close the issue? If it's because I didn't answer yesterday, I have little free time during the week, I don't connect often.

No no its not because of that. I understand life can be busy and truly appreciate all you have done. 
I wouldnt expect you to be on standby for me lol

I realized my settings on my media center were more at fault then the app. I was using higher DPI to have bigger text. When I switched to default settings problem pretty much solved.

If the menus were a bit shorter or scroll-able this would still be a great plus.


However I closed the issue as seeing my system settings were more the culprit.
